extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -229,6 +369,70 @@ class MultipleBackendTask extends ReplicateObject {
         });
     }
 
+    /**
+     * Get byte ranges for an object of the given content length, such that the
+     * range count does not exceed 1024 elements.
+     * @param {Number} contentLength - The content length of the whole object
+     * @return {Array} The array of byte ranges.
+     */
+    _getRanges(contentLength) {
+        const ranges = [];
+        let start = 0;
+        let end = 0;
+        for (let i = 0; i < MPU_GCP_MAX_PARTS; i++) {
+            end = start + Math.ceil(contentLength / MPU_GCP_MAX_PARTS) - 1;
```

Comment:
No big deal, but `Math.ceil(contentLength / MPU_GCP_MAX_PARTS)` may be precomputed to save some CPU cycles.

Reply:
Now that I think about it, it may be more natural to round the part size to the nearest power of two (because computers are generally more efficient dealing with those, may make a small difference for GET/PUT while possibly reducing fragmentation of parts when copying the objects around).